src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+
+type Props = {
+  name:string,
+};
+
+export class Clock extends Component<Props, {}> {
+  state = {
+    date: new Date(),
+  };
+
+  timerIdEverySecond = 0;
+
+  componentDidMount() {
+    this.timerIdEverySecond = window.setInterval(() => {
+      const date = new Date();
+
+      this.setState({ date });
+      // eslint-disable-next-line no-console
+      console.info(this.timeStyle(date));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProp:Props) {
+    const { name } = this.props;
+
+    if (name !== prevProp.name) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevProp.name} to ${name}`);
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerIdEverySecond);
+  }
+
+  timeStyle = (time:Date) => time.toUTCString().slice(-12, -4);
+
+  render() {
+    const {
+      date,
+    } = this.state;
```

Comment:
it's redundant to move the property to the new line if you have only one in state
```suggestion
    const { date } = this.state;
```